catalog/admin/admin-poller-service/src/test/java/org/codice/ddf/catalog/admin/poller/AdminPollerTest.java

```diff
@@ -148,13 +94,20 @@ public void testAllSourceInfo() {
 
         assertThat(sources.get(0), not(hasKey("configurations")));
         assertThat(sources.get(1), hasKey("configurations"));
+        List<Map<String, Object>> configurations = (List) sources.get(1)
+                .get("configurations");
+
+        assertThat(configurations, hasSize(1));
+        Map<String, Object> configurationMap = configurations.get(0);
+        assertThat(configurationMap, hasKey("operation_actions"));
+        assertThat(configurationMap, hasKey("report_actions"));
 
-        // Assert that the password value was changed from "secret" to "password".
-        String password =
-                (String) ((Map<String, Object>) ((List<Map<String, Object>>) sources.get(1)
-                        .get("configurations")).get(0)
-                        .get("properties")).get("password");
-        assertThat(password, is(equalTo("password")));
+        List<Map<String, Object>> operationActions =
+                (List) configurationMap.get("operation_actions");
+        assertThat(operationActions, hasSize(2));
```

Comment:
We should use `hasItems()` instead of `hasSize()` here to make sure the right actions are in the list.